pages/linux/terminator.md

```diff
@@ -0,0 +1,19 @@
+# terminator
+
+> Multiple GNOME terminals in one window.
```

Comment:
How about - `Arrange multiple GNOME terminals in one window` ?

Reply:
I thought it would be better if we use the original description of it in `man` page. don't you think?

Reply:
Not always. `man` page descriptions are terse and meant to be comprehensive. Our motto is to make things simple and easy to understand for beginners. In that context, it is always good to have descriptions which are absolutely crystal clear. Hence, my suggestion.

If we always copy from the man pages, the user might as well do `man terminator` instead of `tldr terminator` :laughing: